lib/sidekiq_flow/client.rb

```diff
@@ -95,7 +93,9 @@ def connection_pool
       private
 
       def adapters
-        @adapters ||= [SidekiqFlow::Adapters::LegacyStorage]
+        #@adapters ||= [SidekiqFlow::Adapters::SetStorage,SidekiqFlow::Adapters::LegacyStorage]
+        @adapters ||= [SidekiqFlow::Adapters::SetStorage]
```

Comment:
let's use `[SidekiqFlow::Adapters::SetStorage,SidekiqFlow::Adapters::LegacyStorage]` for now.

Reply:
resolved but not changed?

Reply:
Yeah, mb, forgot te remove it, but right now I need only the new adapter for the specs. If I keep the legacy one, it will mess with the redis key counts in the specs

Reply:
> it will mess with the redis key counts in the specs

Yeah, I guess we need to replace/refactor those count specs as well to check for actual keys/values.